bot/cogs/utility/nickrequest.py

```diff
@@ -17,11 +17,19 @@ def __init__(self, bot: Bot):
         self.guild = None
 
     @commands.command(name="nick")
-    @commands.cooldown(1, command_timeout, commands.BucketType.user)
-    async def nick(self, ctx: commands.Context, *, nick: str):
+    # @commands.cooldown(1, command_timeout, commands.BucketType.user)
+    async def nick(self, ctx: commands.Context, *, nick):
         acc_channel = self.bot.get_channel(accept_channel_id)
+        nick = str(nick)
+        nick.replace('"', '\"')
+        nick.replace("'", "\'")
```

Comment:
Replacing but not reassigning (i.e. `nick = nick.replace(...)`).
The replacements wouldn't do anything even if reassigned, did you mean to do something like: `('"', '\\"')`?